tests/testthat/test-widerect.R

```diff
@@ -67,14 +70,32 @@ wb.facets <-
 
 info <- animint2HTML(wb.facets)
 
-test_that("widerect renders a <rect> for every year", {
-  node.set <-
-    getNodeSet(info$html,
-               '//g[@class="geom6_widerect_ts"]//g[@class="PANEL1"]//rect')
-  expect_equal(length(node.set), nrow(years))
-  for(node in node.set){
-    sizes <- as.numeric(xmlAttrs(node)[c("height", "width")])
-    expect_true(all(sizes > 0))
+dasharrayPattern <-
+  paste0("stroke-dasharray:",
+         "(?<value>.*?)",
+         ";")
+
+rect.xpaths <-
+  c('//g[@class="geom6_widerect_ts"]//g[@class="PANEL1"]//rect',
+    '//g[@class="geom1_tallrect_ts"]//g[@class="PANEL4"]//rect')
+
+test_that("wide/tallrect renders a <rect> for every year", {
+  for(rect.xpath in rect.xpaths){
+    node.set <- getNodeSet(info$html, rect.xpath)
+    expect_equal(length(node.set), nrow(years))
+    style.list <- list()
+    for(node.i in seq_along(node.set)){
+      node <- node.set[[node.i]]
+      style.list[[node.i]] <- a.vec[["style"]]
+      a.vec <- xmlAttrs(node)
+      sizes <- as.numeric(a.vec[c("height", "width")])
+      expect_true(all(sizes > 0))
+    }
+    style.vec <- do.call(c, style.list)
+    dash.mat <- str_match_perl(style.vec, dasharrayPattern)
+    dash.table <- table(dash.mat[, "value"])
+    ## There should be 2 unique values of stoke-dasharray.
+    expect_equal(length(dash.table), 2)
```

Comment:
This new test makes sure that the aes(linetype=status) in the geom_tallrect and geom_widerect gets translated to 2 different values of stroke-dasharray in the <rect> elements.